custom/icds_reports/models.py

```diff
@@ -18,6 +18,12 @@
 )
 
 
+class AggregateSQLProfile(models.Model):
+    script_name = models.TextField()
+    line_num = models.PositiveSmallIntegerField()
```

Comment:
This should be a text as well. Line numbers can change and don't mean much, but having a name will convey much more meaning

Reply:
Good idea. Fixed it